### PR TITLE
update runtime

### PR DESCRIPTION
**Pull request by dgcampea**

### CODE REVIEW

File: io.github.Hexchat.Plugin.OTR.json [hunk 1]
```diff
@@ -35,21 +35,22 @@
...
         "appstream-compose --basename=io.github.Hexchat.Plugin.OTR --prefix=${FLATPAK_DEST} --origin=flatpak io.github.Hexchat.Plugin.OTR"
       ],
       "sources": [
         {
-          "type": "archive",
-          "url": "https://github.com/TingPing/hexchat-otr/releases/download/0.2.2/hexchat-otr-0.2.2.tar.xz",
-          "sha256": "e18425f4b20447caddf64a9079b8750200100e158a03575d89457bb6d8f8d5f1"
+          "type": "git",
+          "url": "https://github.com/TingPing/hexchat-otr.git",
+          "branch": "master"
```
Comment (TingPing) [inline io.github.Hexchat.Plugin.OTR.json]: This needs to be a specific commit.

**Comment by TingPing** [comment]
bot, build

**Comment by TingPing** [comment]
I imagine the repo just needs time for publishing to go through and be tried later.

Or maybe you need to modify pkg-config path (but I assumed you tested it).

**Comment by dgcampea** [comment]
bot, build

**Comment by dgcampea** [comment]
strange, I remember it built locally without any additional changes

**Pull request merged**